app/services/analytics_events.rb

```diff
@@ -32,7 +32,8 @@ def account_reset(
     mfa_method_counts: nil,
     pii_like_keypaths: nil,
     error_details: nil,
-    email_addresses: nil
+    email_addresses: nil,
+    **extra
```

Comment:
If this is going to become a pattern, is there an easy way to log what the extra attributes are? That way it's easy to hunt them down later.

Reply:
maybe we could add something like `undocumented_keys: extra.keys` to the payload?